.gitpod.yml

```
@@ -66,3 +66,7 @@ github:
     addBadge: false
     # add a label once the prebuild is ready to pull requests (defaults to false)
     addLabel: false
+
+vscode:
+  extensions:
+    - dbaeumer.vscode-eslint@2.1.5:9Wg0Glx/TwD8ElFBg+FKcQ==
```

```suggestion
    - dbaeumer.vscode-eslint@2.1.5:9Wg0Glx/TwD8ElFBg+FKcQ==
    - esbenp.prettier-vscode@4.7.0:kgoYSh1LwTixKtJtQWtodQ==
```
If we're going to add Eslint, adding Prettier makes sense.  Otherwise formatting a file can create issues as far as Eslint is concerned.

---

I didn't get Prettier to work in Gitpod. @ojeytonwilliams , did you get it to work?

---

It seemed to be working when I tried this config.  If you commit this suggestion, you can try it yourself.  If it doesn't work, we can always revert.

---

Fair point. Let's go ahead then....